csrc/velox/functions/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@
 # LICENSE file in the root directory of this source tree.
 
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${TORCH_CXX_FLAGS}")
```

Comment:
hope Velox CXX Flags is compatible with Torch CXX flags... (otherwise will be a nightmare)

Velox CXX flags: (https://github.com/facebookincubator/velox/blob/1bdf9fc2a0ba7e9351cc6d092aa5c0e4e0f78708/CMakeLists.txt#L125-L127):

```
-mavx2 -mfma -mavx -mf16c -mbmi2 -march=native
```